File: Tutorial/Onboarding/C#/Onboarding.cs
using System;

class Player
{
    static void Main()
    {
        while (true)
        {
            string enemy1Name   = Console.ReadLine();
            int enemy1Distance  = int.Parse(Console.ReadLine());
            string enemy2Name   = Console.ReadLine();
            int enemy2Distance  = int.Parse(Console.ReadLine()); 

            Console.WriteLine((enemy1Distance < enemy2Distance) ? enemy1Name : enemy2Name);
        }
    }
}

File: Tutorial/Onboarding/Java/Onboarding.java
import java.util.*;

class Player {

    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        while (true) {
            String enemy1Name   = in.next();
            int enemy1Distance  = in.nextInt();
            String enemy2Name   = in.next();
            int enemy2Distance  = in.nextInt();

            System.out.println(enemy1Distance < enemy2Distance ? enemy1Name: enemy2Name);
        }
    }
}